src/main/flight/pid.c

```diff
@@ -611,7 +612,14 @@ static void NOINLINE pidApplyFixedWingRateController(pidState_t *pidState, fligh
     const float newFFTerm = pidState->rateTarget * pidState->kFF;
 
     // Calculate integral
-    pidState->errorGyroIf += rateError * pidState->kI * dT;
+    // Freeze yaw Iterm when bank angle is above threshold to avoid rudder counteracting turns
+    float bankAngle = DECIDEGREES_TO_DEGREES(attitude.values.roll);
+    if (fabsf(bankAngle) > pidProfile()->fixedWingYawItermBankLimit && axis == FD_YAW && !FLIGHT_MODE(AUTO_TUNE)) {
+        pidState->errorGyroIf += 0;
```

Comment:
@avsaase the same thing can be applied by enabling `pidState->itermLimitActive` in `checkItermLimitingActive` function. This would require making `checkItermLimitingActive` axis aware but as a result would be cleaner than current implementation and would not require any changes to `pidApplyFixedWingRateController`

Reply:
If I understand correctly `applyItermLimiting` does the actual I-term limiting if I do it like that, correct? But that function stops the I-term from growing, but not from shrinking. I'm not sure if that makes sense in this case.